public/view_thread.php

```diff
@@ -1,18 +1,19 @@
-<?php require('header.php'); ?>
-
-<?php 
-
-require('database.php');
+<?php
+	session_start();
+?>
+<?php require('database.php');
```

Comment:
I think we need to combine these into 1 php tag if we want the redirect to work on line 27